modules/codemod/README.md

```diff
@@ -13,13 +14,18 @@ Commands (transforms):
   canvas-kit-codemod v5 [path]  Canvas Kit v4 > v5 migration transform
 
 Options:
-  -v, --version  Show version number                                   [boolean]
-  -h, --help     Show help                                             [boolean]
+  --ignore-config,  Ignore files if they match patterns sourced from a config file [string]
+  --ignore-pattern, Ignore files that match a provided glob expression             [string] [default: **/node_modules/**]
+  --verbose,        Show more information about the transform process              [number] [choices: 0, 1, 2] [default: 0]
+  -v, --version     Show version number                                            [boolean]
+  -h, --help        Show help                                                      [boolean]
 ```
 
-> Note: These codemods only work on .js, .jsx, .ts, and .tsx extensions. You may need to make some manual changes in other file types (.json, .mdx, .md, etc.).
+> Note: These codemods only work on .js, .jsx, .ts, and .tsx extensions. You may need to make some
+> manual changes in other file types (.json, .mdx, .md, etc.).
```

Comment:
why is `in` highlighted differently here?

Reply:
That's GitHub trying to syntax highlight correctly and making a mistake. It thinks it's a keyword.